### ISSUE TITLE
Unable to build any linux ARM binary

### ISSUE THREAD

**Issue by stevielavern** [frida/frida-core#190]
I managed to build the linux ARM SDK. 
However when trying to build any linux ARM binaries (server-arm, inject-arm or gadget-arm), I stumble upon this error:
```
$ make server-arm
../../../frida-core/src/linux/frida-helper-service-glue.c: In function ‘frida_inject_instance_emit_payload_code’:
../../../frida-core/src/linux/frida-helper-service-glue.c:1532:5: warning: implicit declaration of function ‘EMIT_MOV’; did you mean ‘EMIT_MOVE’? [-Wimplicit-function-declaration]
     EMIT_MOV (R3, R0);
     ^~~~~~~~
     EMIT_MOVE
../../../frida-core/src/linux/frida-helper-service-glue.c:1532:15: error: ‘R3’ undeclared (first use in this function)
     EMIT_MOV (R3, R0);
               ^~
../../../frida-core/src/linux/frida-helper-service-glue.c:1532:15: note: each undeclared identifier is reported only once for each function it appears in
../../../frida-core/src/linux/frida-helper-service-glue.c:1532:19: error: ‘R0’ undeclared (first use in this function); did you mean ‘R3’?
     EMIT_MOV (R3, R0);
                   ^~
                   R3
../../../frida-core/src/linux/frida-helper-service-glue.c: In function ‘_frida_helper_service_do_spawn’:
../../../frida-core/src/linux/frida-helper-service-glue.c:389:5: warning: ignoring return value of ‘chdir’, declared with attribute warn_unused_result [-Wunused-result]
     chdir (old_cwd);
     ^~~~~~~~~~~~~~~
[12/17] Compiling C object 'src/frida-helper-service@sta/meson-generated_linux_frida-helper-types.c.o'.
ninja: build stopped: subcommand failed.
Makefile.linux.mk:363: recipe for target 'build/tmp-linux-arm/frida-core/.frida-helper-and-agent-stamp' failed
make[1]: *** [build/tmp-linux-arm/frida-core/.frida-helper-and-agent-stamp] Error 1
make[1]: Leaving directory '/home/user/frida'
Makefile:2: recipe for target 'server-arm' failed
make: *** [server-arm] Error 2
```
Looks like a typo in `frida-helper-service-glue.c`. When changing `EMIT_MOV` to `EMIT_MOVE`, I finally get another error:

```
. build/frida-meson-env-linux-x86_64.rc && /home/user/frida/releng/ninja-linux-x86_64 -C build/tmp-linux-arm/frida-core install
ninja: Entering directory `build/tmp-linux-arm/frida-core'
ninja: error: '../../../frida-core/tests/labrats/forker-linux-arm', needed by 'tests/labrats/sleeper-linux-arm', missing and no known rule to make it
Makefile.linux.mk:332: recipe for target 'build/frida-linux-arm/lib/pkgconfig/frida-core-1.0.pc' failed
make: *** [build/frida-linux-arm/lib/pkgconfig/frida-core-1.0.pc] Error 1
```

Any idea of what's wrong here?
FYI, building server-64 (linux x86-64) worked perfectly out-of-the box.

**Comment by stevielavern** [comment]
Actually this seems to be related to tests only. Commenting out the last line of `frida-core/meson.build` works:

```
$ tail -1 frida-core/meson.build
#subdir('tests')
$ make server-arm
[...]
$ ls build/frida-linux-arm/bin/
frida-inject  frida-server
```

**Comment by oleavr** [comment]
> Looks like a typo in frida-helper-service-glue.c

Correct! Would you mind opening a PR?

> Any idea of what's wrong here?

We need to build the labrats for linux-arm – just add the it to `tests/labrats/Makefile.linux.mk`, then `make -f tests/labrats/Makefile.linux.mk` and open a PR with the resulting binaries. The reason we check these in is so they don't change underneath us, so tests have a fixed point of reference.

**Comment by stevielavern** [comment]
This had been fixed up to frida 12.2.30, but frida-core commit 4da3d2d687fa862f9 deleted `spawner-linux-arm` and now the Linux-ARM SDK fails to build again with:
```
ninja: error: '../../../frida-core/tests/labrats/spawner-linux-arm', needed by 'tests/labrats/sleeper-linux-arm', missing and no known rule to make it
```

I unfortunately do not have much resources to open a PR. Would it be possible to either re-add the missing file, or add a way to skip tests during build?

Thanks!

**Comment by stevielavern** [comment]
Closing, as the #232 PR make it possible to skip tests.
